packages/core/src/sass/app/modules/cashier.scss

```diff
@@ -912,8 +912,8 @@
         }
     }
     &__drop-down {
-        min-width: calc(calc(100% - calc(16px + 3.2rem)) / 2);
-        width: auto;
+        min-width: calc(calc(100% - calc(16px + 3.2rem)) / 2) !important;
```

Comment:
`!important` looks hacky, Why is this being overridden?

Reply:
It has to override a styling with higher specificity.

Reply:
@reza-fs are you satisfied with this?

@mahdiaryayi-fs of course `!important` is used to override another styling with higher specificity, but why is this needed? as in, can the specificities (in this file and the other file) be written in a way that they all work without `!important`?

Reply:
@reza-fs @mahdipourziaei-binary I checked this again. removing the `!important` from CSS doesn't make any bug this time. I think the order of styles is changed in other cards and fixed this issue. I removed them for now but since this is a Safari related bug and I am on a Linux machine, so I can not test it myself on Safari in WFH days!